rasa/engine/storage/storage.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
```

Comment:
Shall we call this file `interface.py` to match the `runner/interface.py`? Or should I rename the runner?

Reply:
I didn't call it interface because it also contains `ModelMetadata` and some constants 🤔 `interface` sounds like it could only contain interfaces. Maybe a more general name like repeating the package name hence is a little bit better